integration-tests/aws2/src/main/resources/application.properties

```diff
@@ -20,15 +20,8 @@
 camel.context.name = quarkus-camel-example
 
 #
-# Camel :: AWS SQS
+# Camel :: AWS SNS
 #
-camel.component.aws2-sqs.access-key={{env:AWS_ACCESS_KEY}}
-camel.component.aws2-sqs.secret-key={{env:AWS_SECRET_KEY}}
-camel.component.aws2-sqs.region={{env:AWS_REGION}}
-
-#
-# Camel :: AWS S3
-#
-camel.component.aws2-s3.access-key={{env:AWS_ACCESS_KEY}}
-camel.component.aws2-s3.secret-key={{env:AWS_SECRET_KEY}}
-camel.component.aws2-s3.region={{env:AWS_REGION}}
\ No newline at end of file
+camel.component.aws2-*.access-key={{env:AWS_ACCESS_KEY}}
```

Comment:
How is this asterisk key matched? The S3 extension checks both `camel.component.aws2-s3` and `camel.component.aws2-*` and uses whichever of the two is set?

Reply:
This is a feature introduced in camel 3.2.0, in this case it will set the main options for all aws2-* components

Reply:
Thanks, interesting.

Reply:
Here is the related issue
https://issues.apache.org/jira/browse/CAMEL-14598